README.md

```diff
@@ -35,6 +34,13 @@ MeasurementLink Support for Python (`ni-measurementlink-service`) is a Python fr
 
 ---
 
+## Documentation
+
+- [MeasurementLink Manual](https://www.ni.com/docs/en-US/bundle/measurementlink)
```

Comment:
@nstokes-nati Is this the right terminology? And is this a stable URL that won't change?

Reply:
Yes, that's fine terminology. And yes, that's the best format for a direct link and I don't think this really merits a redirect.